### PR TITLE
[WIP] Fix in-pixel metadata keys

### PR DESCRIPTION
**Pull request by sperand-io**
per @jtotoole's example here: https://gist.github.com/jtotoole/1b0f14088dd2d5ba78d54fe942fccd83

### CODE REVIEW

**Comment by sperand-io** [comment]
@hankim813

File: lib/index.js [hunk 1]
```diff
@@ -67,15 +68,18 @@ Parsely.prototype.page = function(page) {
...
+      pub_date: aliasedProps.pub_date || aliasedProps.created,
+      title: aliasedProps.title || page.title(),
+      tags: aliasedProps.tags,
+      authors:  aliasedProps.authors,
+      link: aliasedProps.link || page.url(),
+      page_type: aliasedProps.page_type || 'post'
     };
 
+    if (!is.array(metadata.authors)) metadata.authors = [metadata.authors];
+
```
Comment (jtotoole) [inline lib/index.js]: the `tags` field takes an array in the same manner as `authors`, e.g. `"tags": ["yeezus", "college dropout", "808s and heartbreak"]`

File: test/index.test.js [hunk 2]
```diff
@@ -158,17 +163,20 @@ describe('Parsely', function() {
...
-          dateCreated: 'running back',
-          headline: 'loyal',
-          keywords: 'put on',
-          creator: 'm.A.A.d city',
-          url: 'Reminder'
+          section: 'father stretch my hands pt.1',
+          image_url: 'started from the bottom',
+          pub_date: 'running back',
+          title: 'loyal',
+          tags: 'put on',
```
Comment (jtotoole) [inline test/index.test.js]: should be ['put on']

**Comment by jtotoole** [comment]
thanks @sperand-io! added a note re: tag formatting but otherwise looks good to me. appreciate you jumping on this

**Pull request merged**